resolvers/query.js

```diff
@@ -110,5 +137,36 @@ module.exports = {
     },
     user(_, { id }) {
         return User.findById(id);
+    },
+    async timeSheetRecordsToCSV(_, { userId, projectId, startDate, endDate }) {
+        const records = await TimesheetRecord.findAll({
+            raw: true,
+            where: {
+                userId: userId,
+                projectId: projectId,
+                date: {
+                    [Op.between]: [startDate, endDate]
+                }
+            },
+            include: [
+                {
+                    model: Project,
+                    as: 'project',
+                    attributes: ['name'],
+                    where: {
```

Comment:
Seems excessive, re-check if the result is the same without this `where` clause

Reply:
fix